integration-authentication-api/src/main/java/org/symphonyoss/integration/authentication/api/model/JwtPayload.java

```diff
@@ -0,0 +1,98 @@
+package org.symphonyoss.integration.authentication.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.Date;
+import java.util.Objects;
+
+/**
+ * Holds the Symphony's user authentication data as a JWT.
+ *
+ * Created by campidelli on 15/08/17.
+ */
+public class JwtPayload {
+
+  @JsonProperty("aud")
+  private String applicationId;
+
+  @JsonProperty("iss")
+  private String companyName;
+
+  @JsonProperty("sub")
+  private String userId;
+
+  @JsonProperty("exp")
+  @JsonDeserialize(using = UnixTimestampDeserializer.class)
+  private Date expirationDate;
```

Comment:
Because we need Date to work with, otherwise we should have to create a Date object all the time we have to access this property. The custom deserializer was made to convert from seconds (JWT pattern) to milliseconds, and not because it's a Date.